contracts/loadtester/LoadTester.sol

```diff
@@ -5,6 +5,7 @@ contract LoadTester {
     uint256 callCounter;
     uint256 loopCounter;
     bytes[] public dumpster;
+    string public constant RevertErrorMessage = "RevertErrorMessage"; 
```

Comment:
I think this message is a little too generic. It might be nice to have something a little more distinctive.
In this message it doesn't jump out that `RevertErrorMessage` is the message. It looks more like a label
![image](https://github.com/maticnetwork/polygon-cli/assets/429588/ff19e8f2-9235-4249-ab28-1009882affc7)

Reply:
haha true, updated